fastlane.gemspec

```diff
@@ -64,8 +64,11 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'faraday', '~> 0.9' # Used for deploygate, hockey and testfairy actions
   spec.add_dependency 'faraday_middleware', '~> 0.9' # same as faraday
 
-  # Lock `activesupport` (transitive depedency via `xcodeproj`) to keep supporting system ruby
-  spec.add_dependency 'activesupport', '< 5'
+  if ENV['RUBY_VERSION'] < '2.2.2'
```

Comment:
this doesn't work, when released to RubyGems, the spec is re-serialized and the conditional will be evaluated on the machine that runs `gem build` and `gem push`